### PR TITLE
Update foreman_pipeline to 0.0.11

### PR DESCRIPTION
**Pull request by xprazak2**
New release with updated dependencies.

### CODE REVIEW

**Comment by xprazak2** [comment]
Seems like build succeeds, but my assets cannot be found when the package actually installs. Will do some digging.

**Comment by xprazak2** [comment]
Issue with assets resolved, everything seems to be ok now.

File: rubygem-foreman_pipeline/rubygem-foreman_pipeline.spec [hunk 1]
```diff
@@ -11,54 +11,39 @@
 
 %global gem_name foreman_pipeline
 %global foreman_dir /usr/share/foreman
-%global scl_rake /usr/bin/%{?scl:%{scl_prefix}}rake
+%global foreman_bundlerd_dir %{foreman_dir}/bundler.d
```
Comment (ehelms) [inline rubygem-foreman_pipeline/rubygem-foreman_pipeline.spec]: This shouldn't be needed as the macro handles it. Do you get errors if you don't define it here?

File: rubygem-foreman_pipeline/rubygem-foreman_pipeline.spec [hunk 2]
```diff
@@ -11,54 +11,39 @@
...
 
-%define rubyabi 1.9.1
+%global scl_rake /usr/bin/%{?scl:%{scl_prefix}}rake
+%global scl_ruby_bin /usr/bin/%{scl:%{scl_prefix}}ruby
```
Comment (ehelms) [inline rubygem-foreman_pipeline/rubygem-foreman_pipeline.spec]: This shouldn't be needed as the macro handles it. Do you get errors if you don't define it here?

**Comment by xprazak2** [comment]
I removed `%global` definitions for rake, bundler.d and ruby, it seems be ok.

**Comment by ehelms** [comment]
Thanks @xprazak2 ! I will build it into nightly and 3.0

**Pull request merged**